styles/components/badge.css

```diff
@@ -0,0 +1,54 @@
+.badge-container {
+  position: relative;
+  width: fit-content;
+  height: fit-content;
+}
+
+.badge {
+  position: absolute;
+  top: 0;
+  right: 0;
+  border-radius: 50%;
+  font-size: 0.7rem;
+  padding: 0.5rem;
+  background: var(--primary-color);
+  color: white;
+}
+
+.badge-md {
+  font-size: 1rem;
+}
+
+.badge-lg {
+  font-size: 1.3rem;
+}
+
+.badge-status {
+  bottom: 0.8rem;
+  right: 0.6rem;
+  top: auto;
+  left: auto;
+  height: 0.1rem;
+  width: 0.1rem;
+  border: 3px solid white;
+}
+
+.badge-status.badge-online {
```

Comment:
I suppose ".badge-online" would have been sufficient

Reply:
Hey! thanks for the review. I have made the suggested change. For the position of badge, i tried to lower its position but then it does not look good with larger avatars. So made a trade-off between those two positions.